packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -118,8 +118,32 @@ def invoke_detect_ip():
 
 # Add backend specific arguments
 exhibitor_backend = get_var_assert_set('EXHIBITOR_BACKEND')
-if zookeeper_cluster_size == 1:
+if exhibitor_backend == 'STATIC':
+    print("Exhibitor configured for static ZK ensemble")
+    # In case of a static Exhibitor backend DC/OS configuration we can check
+    # that the value for the hostname set by invoking the ip-detect script is
+    # indeed included in the master list.
+    # https://jira.mesosphere.com/browse/COPS-3485
+    exhibitor_staticensemble = get_var_assert_set('EXHIBITOR_STATICENSEMBLE')
+    master_ips = [i.split(':')[1] for i in exhibitor_staticensemble.split(',')]
+    if detected_ip not in master_ips:
+        message = (
+            "ip-detect returned {master_ip}. "
```

Comment:
There is another instance of error printing in this script with prefix `ERROR: `. Maybe we should be consistent.

Reply:
Yes, good point. That's another 3 hours CI round-trip :dagger: